.github/workflows/build.yaml

```diff
@@ -33,21 +34,32 @@ jobs:
           eval ${{ matrix.env }}
           image="ros-abuild:${ALPINE_VERSION}-${ROS_DISTRO}"
 
-          git clone --depth=1 -b ${ROS_DISTRO}-devel https://github.com/ros/rospack /tmp/rospack
+          if [[ ${ROS_DISTRIBUTION_TYPE} == "ros1" ]]; then
+            pkg_name="rospack"
+            git clone --depth=1 -b ${ROS_DISTRO}-devel https://github.com/ros/${pkg_name} /tmp/${pkg_name}
+          else
+            git clone --depth=1 -b ${ROS_DISTRO} https://github.com/ros2/geometry2 /tmp/geometry2
+            pkg_name="tf2_msgs"
```

Comment:
I think it's better to use a package with builds and unit tests.
I selected `rospack` to test c++ build with unit tests (without too many deps)

Reply:
I changed to `tf2_eigen_kdl`. It has c++ code and some tests.

Reply:
`rospack` have no dependency to other ROS packages except catkin to make it possible to be tested when package repository is not fully ready yet.
Is there a ros2 package like this?

Maybe the best solution is to have a simple dummy package with `<buildtool_depend>cmake</buildtool_depend>`

Reply:
What about https://github.com/ament/ament_index/tree/rolling/ament_index_cpp? It does not seem to have dependency on other ROS package but uses `<buildtool_depend>ament_cmake</buildtool_depend>` though.

Reply:
looks good